dist/ResizeObserver.js

```diff
@@ -110,6 +110,9 @@
   var transitionKeys = ['top', 'right', 'bottom', 'left', 'width', 'height', 'size', 'weight'];
   // Check if MutationObserver is available.
   var mutationObserverSupported = typeof MutationObserver !== 'undefined';
+  // Get the native implementation of HTMLElement.attachShadow
+  var origAttachShadow = (isBrowser && HTMLElement.prototype.attachShadow &&
```

Comment:
Ah, right. We don't have to care about polyfilled shadow dom because it doesn't modify mutation observers to not see into the shadow dom, yeah?

Reply:
Exactly, yes.